README.md

```diff
@@ -166,6 +168,17 @@ simply `npm install` and `npm test`.
     provisions of this method, which you cannot possibly
     obtain in legacy engines.
 
+### Example of applying ES compatability shims in a browser project
+
+```html
+<script src="es5-shim.js"></script>
+<script src="es5-sham.js"></script>
+<script src="json3.js"></script>
```

Comment:
There is no `cdnjs` for `es7-shim` yet. And `cdnjs` does not have a "latest" option unfortunately (not that I could find), it's by release version.